Backend/app/dashboard/CommonAPIs/src/Dashboard/RiderPlatform/Ride.hs

```diff
@@ -229,3 +229,39 @@ instance HideSecrets MultipleRideSyncReq where
 validateMultipleRideSyncReq :: Validate MultipleRideSyncReq
 validateMultipleRideSyncReq MultipleRideSyncReq {..} = do
   validateField "rides" rides $ UniqueField @"rideId"
+
+---------------------------------------------------------
+-- Ticket Ride List--------------------------------------
+
+type TicketRideListAPI =
```

Comment:
Types of these APIs are exactly same. Why don't you create it inside Dashboard/Common/Ride.hs and reexport from here?

Reply:
RideInfo which is present in response types of both the apis is having different values like personId = Id Driver and Id customer  , status is different so they have different types